data/human/human missions.txt

```diff
@@ -4759,3 +4759,200 @@ mission "Dangerous Games 2"
+			choice
+				`	(Put your gun away.)`
+					goto away
+				`	(Shoot.)`
+			action
+				set "Shot Core"
+			`	You advance on the Core and start firing at it. Church begs for you to stop until his voice cuts off. Lights on the Core turn from white to red, then fade. Riley hasn't moved, stunned.`
+			`	You head back to your ship, leaving behind a father survived by his daughter; no more and no less than nature intended.`
+				decline
+			label away
+			`	Riley relaxes slightly once she sees your gun put back in its holster. After the shock has subsided, she finally speaks. "Leave us alone," she says, her voice still trembling. Realizing your time here is up, you exit the hospital and make your way to the spaceport, hoping that that scene did not attract unwanted notice.`
+			choice
+				`	(Turn the Core in for a bounty.)`
+					goto bounty
+				`	(Go back to your ship.)`
+			`	Deciding to leave this planet behind you, you start up your ship and make your way back to the stars.`
+					decline
+			label suggest
+			`	"It's a nice idea," she smiles, "but I'm not a starfarer. The big city is where I'll end back up, eventually."`
+			label goodbye
+			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
+			`	You can feel her smile on your back until the door closes behind you. Despite her happiness, you wonder if her attachment to the AI is healthy.`
+				decline
+			label death
+			`	Riley is stunned for a moment. "You don't know that," she whispers. "Sure, he's a lot more forgetful, and doesn't seem as emotional, but... maybe it just takes some time for the system to warm up. Besides, you don't know what he means to me. What he's done for me. Even if this isn't really my father, I can't turn my back on him."`
+			choice
+				`	(Say goodbye.)`
+				`	(Turn the Core in for a bounty.)`
+					goto bounty
+				`	(Draw your gun and destroy the Core).`
+					goto destroy
+			`	Respecting Riley's request to stay with her father, you make your farewells and head for the door.`
+			`	Despite her previous happiness, you notice that her disposition has become more muted after what you said. You wonder if she has begun to doubt whether her attachment to the AI is healthy.`
+				decline
+			label bounty
+			action
+				payment 100000
+			`	You say your goodbyes and leave. From across the street, you make a call to the local Deep Security office and report what you saw. A slick voice on the other side of your commlink responds.`
+			`	"Ah. Mr. Church has been on our watchlist for some time. This is not exactly surprising. However, we've been unable to get decisive proof on any illegal activity. This information, assuming it is correct, will be a great boon to us. Thank you for the tip." A few minutes pass as you watch their forces descend on the house.`
+			`	Your bank account notifies you that 100,000 credits have been added to your account. Turning away, you head back to your ship.`
```

Comment:
Instead of having the cops apparently be on DEFCON 1 for this one old guy, perhaps have the raid take place off-screen.
```suggestion
			`	"Ah. Mr. Church has been on our watchlist for some time. This is not exactly surprising. However, we've been unable to get decisive proof on any illegal activity. This information, assuming it is correct, will be a great boon to us. Thank you for the tip."`
			`	By the time you arrive back at the spaceport, your bank account notifies you that <payment> have been added to your account.`
```